Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Usamos la imagen base elegida
+FROM alpine:latest
+
+RUN adduser --disabled-password user_test \
+    && mkdir -p /home/user_test/app \
+    && chown -R user_test /home/user_test/app
+
+WORKDIR /home/user_test/app
+
+# Copiamos el package.json al directorio principal para poder instalar las dependencias
+COPY package.json pnpm-lock.yaml ./
+
+RUN wget -qO /bin/pnpm "https://github.com/pnpm/pnpm/releases/latest/download/pnpm-linuxstatic-x64" \
+    && chmod +x /bin/pnpm \
+    && apk add nodejs
```

Comment:
¿Qué versión de `node` instala esto?

Reply:
La última version de `node` disponible en el gestor de paquetes de Alpine, concretamente la 18.12.1

Reply:
Por favor, ¿podéis entender de una vez por todas que cuando se hace una objeción no es para iniciar una conversación, sino para que realicéis algún cambio, aquí o en la documentación?